web_widget_float_formula/static/src/js/web_widget_float_formula.js

```diff
@@ -0,0 +1,92 @@
+/**
+*    Copyright GRAP
+*    Copyright 2016 LasLabs Inc.
+*    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+**/
+
+odoo.define('web.web_widget_float_formula', function(require) {
+    "use strict";
+
+    var core = require('web.core');
+    var basic_fields = require('web.basic_fields');
+    var field_utils = require('web.field_utils');
+    var NumericField = basic_fields.FieldFloat;
+
+    NumericField.include({
+        _formula_text: '',
+
+        events: _.extend({
+            "blur": "_compute_result",
+            "focus": "_display_formula",
+        }, NumericField.prototype.events),
+
+        commitChanges: function() {
+            this._compute_result();
+        },
+
+        _clean_formula_text: function() {
+            this._formula_text = '';
+        },
+
+        _process_formula: function(formula) {
+            try{
+                formula = formula.toString();
+            } catch (ex) {
+                return false;
+            }
+            var clean_formula = formula.toString().replace(/^\s+|\s+$/g, '');
+            if (clean_formula[0] == '=') {
+                clean_formula = clean_formula.substring(1);
+                var myreg = new RegExp('[0-9]|\\s|\\.|,|\\(|\\)|\\+|\\-|\\*|\\/', 'g');
+                if (clean_formula.replace(myreg, '') === '') {
+                    return clean_formula;
+                }
+            }
+            return false;
+        },
+
+        _eval_formula: function(formula) {
+            // Import localization values used to eval formula
+            var translation_params = core._t.database.parameters;
+            var decimal_point = translation_params.decimal_point;
+            var thousands_sep = translation_params.thousands_sep;
+
+            var value;
+            formula = formula.replace(thousands_sep, '').replace(decimal_point, '.');
```

Comment:
In javascript replace only replaces one occurrence. If you have 2 decimal separator in one formula it fails e.g.: =1.2/1.8

This will fix this problem
```
            while (formula.indexOf(decimal_point) !== -1) {
                formula = formula.replace(decimal_point, '.');
            }
```